pkg/vendir/directory/directory.go

```diff
@@ -243,3 +278,17 @@ func maybeChmod(path string, potentialPerms ...*os.FileMode) error {
 
 	return nil
 }
+
+func handleLazySync(oldConfigDigest string, newConfigDigest string, fetchLazyGlobalOverride bool, fetchLazy bool) (bool, bool) {
```

Comment:
We try to keep functions that are only used in a single struct internal to the structure even if they do not use the structure itself. That way we will know the scope in which we expect these functions to run.
With that in mind do you mind adding this function to the Directory struct?

Reply:
Not at all - done

Reply:
Sure - done